src/signUp.js

```diff
@@ -0,0 +1,60 @@
+import { MESSAGE, TEST_AUTH } from './constants/SIGN.js';
+import {
+  emailInput,
+  emailError,
+  pwInput,
+  pwToggle,
+  applyError,
+  resetError,
+  isValidateEmail,
+  isValidatePw,
+  handleClickPwToggle,
+} from './utils/auth.js';
+
+const pwCheckInput = document.getElementById('password-check');
+const pwCheckError = document.getElementById('password-check-error');
+const pwCheckToggle = document.querySelector('.password-check-eye');
+const signUpForm = document.querySelector('#signup-form');
+
+const isDuplicateEmail = () => {
+  if (emailInput.value === TEST_AUTH.EMAIL) {
+    applyError(emailError, MESSAGE.DUPLICATE_EMAIL, emailInput);
+    return true;
+  } else {
+    return false;
+  }
+};
```

Comment:
```suggestion
const isDuplicateEmail = () => {
  if(emailInput.value !== TEST_AUTH.EMAIL) return false;
   applyError(emailError, MESSAGE.DUPLICATE_EMAIL, emailInput);
   return true;
};
```

조금 더 깔끔하게 고쳐보자면 early return을 활용하시면 좋을 것 같습니다!

Reply:
오 early return.... 생각 못하고 있었는데 감사합니다!